plugins/HHAnalyzer.cc

```diff
@@ -324,11 +329,11 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             myllmet.ill = ill;
             myllmet.imet = imet;
             myllmet.isNoHF = met[imet].isNoHF;
-            float dphi = ROOT::Math::VectorUtil::DeltaPhi(ll[ill].p4, met[imet].p4);
+            float dphi = std::abs(ROOT::Math::VectorUtil::DeltaPhi(ll[ill].p4, met[imet].p4));
             myllmet.DPhi_ll_met = dphi;
```

Comment:
In this specific case, I would add the absolute value when filling `myllmet.DPhi_ll_met`, since this `dphi` is used later on